pre/remap_restart/remap_upper.py

```diff
@@ -131,35 +136,37 @@ def remap(self):
        NPE = 5400; nwrit= 6
 
      QOS = "#SBATCH --qos="+config['slurm']['qos']
-     if NPE > 532: QOS = "###" + QOS
-     CONSTR = "#SBATCH --constraint=" + config['slurm']['constraint']
+     TIME ="#SBATCH --time=1:00:00"
+     if NPE > 532: 
+       assert config['slurm']['qos'] != 'debug', "qos should be allnccs"
+       TIME = "#SBATCH --time=12:00:00"
+
+     PARTITION = "#SBATCH --partition=" + config['slurm']['partition']
 
      log_name = out_dir+'/remap_upper_log'
 
      # We need to create an input.nml file which is different if we are running stretched grid
      # First, let's define a boolean for whether we are running stretched grid
      # If we are running with imout of 270, 540, 1080, 1536 or 2160, then we are running stretched grid
-     if imout in [270, 540, 1080, 2160]:
-        stretched_grid = True
-        target_lat = 39.5
-        target_lon = -98.35
-        stretch_fac = 2.5
-     elif imout in [1536]:
-        stretched_grid = True
-        target_lat = 39.5
-        target_lon = -98.35
-        stretch_fac = 3.0
-     else:
-        stretched_grid = False
-
      # If we are running stretched grid, we need to pass in the target lat, lon, and stretch factor
      # to interp_restarts.x. Per the code we use:
      #  -stretched_grid target_lon target_lat stretch_fac
```

Comment:
This is a bit nitpicky, but I think it would be best to use the same order of these three parameters everywhere.  In the comments and the sg001 and sg002 defs in remap_utils.py, we use the order fac/lat/lon.  In the lines below, we use lat/lon/fac, and in the cmd line arg list we use lon/lat/fac.  It's easy to see how someone will accidentally get the order wrong.

Reply:
Mostly addressed in most recent commit https://github.com/GEOS-ESM/GEOS_Util/pull/19/commits/40e43d66028852f696b8fa26189e6dd5ff36af51